python/prairielearn.py

```diff
@@ -117,6 +122,26 @@ def to_json(v):
     If v can be json serialized or does not have a standard type, then it is
     returned without change.
     """
+    if new_np_scalar_encoding and type(v).__module__ == "numpy":
```

Comment:
Numbers are better than bools for new versions. 

```suggestion
    if np_encoding == 2 and type(v).__module__ == "numpy":
```